.github/workflows/release.yml

```diff
@@ -0,0 +1,64 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - mountpoint-s3-[0-9]+.*
+      - mountpoint-s3-[a-z]+-[0-9]+.*
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: taiki-e/create-gh-release-action@v1
+      with:
+        prefix: mountpoint-s3(-[a-z]+)?
+        draft: true
+        # TODO: set it true after we have changlog template reviewed.
+        changelog: false
+        # TODO: set it false after we have changlog template reviewed.
+        allow-missing-changelog: true
+        branch: main
+        title: "Mountpoint for Amazon S3 $version"
```

Comment:
Probably add `v` before the version number, so it will be like `Mountpoint for Amazon S3 v0.0.1`